src/ngraph/runtime/aligned_buffer.hpp

```diff
@@ -46,6 +50,7 @@ class ngraph::runtime::AlignedBuffer
     AlignedBuffer(const AlignedBuffer&) = delete;
     AlignedBuffer& operator=(const AlignedBuffer&) = delete;
 
+    Allocator* m_allocator;
```

Comment:
Turn this into a unique or shared pointer (depending on whether this class will own or not.